src/Controls/tests/DeviceTests/Elements/SearchBar/SearchBarTests.Android.cs

```diff
@@ -17,5 +18,28 @@ Task<string> GetPlatformText(SearchBarHandler handler)
 		{
 			return InvokeOnMainThreadAsync(() => GetPlatformControl(handler).Query);
 		}
+
+		[Fact]
+		public async Task ShowsKeyboardOnFocus()
```

Comment:
Can we move these to a general test class so we don't have this code copied around? 

I think we can create a TextInput set of tests

\TextInput\TextInput.Android.cs

Then you can use a theory with classdata to pass in all the textinput types.